sign.js

```diff
@@ -103,7 +103,7 @@ function verifySignApplicationAsync (opts) {
   })
 
   // Additionally test Gatekeeper acceptance for darwin platform
-  if (opts.platform === 'darwin') {
+  if (opts.platform === 'darwin' && opts['gatekeeper-assess']) {
```

Comment:
Definitely should. Otherwise programmatic default is not equals to CLI and it is strange.